### PR TITLE
Allow selecting heat source process at the ennobling stage.

### PR DESCRIPTION
**Pull request by n1k0**

### CODE REVIEW

File: public/data/processes.json [hunk 1]
```diff
@@ -149,36 +149,6 @@
     "waste": 0.0,
     "alias": null
   },
-  {
-    "name": "Vapeur à partir de gaz naturel (mix de technologies de combustion et d'épuration des effluents gazeux|en sortie de chaudière|Puissance non spécifiée), ES",
```
Comment (n1k0) [inline public/data/processes.json:153]: Note to reviewers, esp. incl. @paulboosz: this Spanish steam process wasn't used anywhere so I took the liberty to remove it.

File: public/data/processes.json [hunk 2]
```diff
@@ -6805,7 +6775,7 @@
   },
   {
     "name": "Vapeur à partir de gaz naturel (mix de technologies de combustion et d'épuration des effluents gazeux|en sortie de chaudière|Puissance non spécifiée), RER",
-    "info": "Energie > Chaleur > Mix moyen",
+    "info": "Energie > Chaleur > Vapeur par énergie primaire",
```
Comment (n1k0) [inline public/data/processes.json:6778]: Note to reviewers, esp. incl. @paulboosz: I also took the liberty to categorize this process the exact same way similar ones are.

**Review by magopian — APPROVED** [review]
Nice work!
Do we plan on having a more fine-tuned heat selection process in the future? Why store and decode continents, when in the end all we want is RER or RSA?

File: src/Data/Textile/Process.elm [hunk 3]
```diff
@@ -60,23 +67,31 @@ type alias WellKnown =
     , passengerCar : Process
     , endOfLife : Process
     , fading : Process
+    , steamGasRER : Process
```
[the remark this answers is not in the document]
Comment (n1k0) [reply src/Data/Textile/Process.elm:70]: It's in the methodology documentation, I don't remember what the acronyms stand for though.

**Comment by n1k0** [comment]
> Do we plan on having a more fine-tuned heat selection process in the future? Why store and decode continents, when in the end all we want is RER or RSA?

The idea is to introduce the concept of geographical zone, which will be used for upcoming features, and also to allow having more specific per-zone process allocations if we ever want to.

**Comment by paulboosz** [comment]
cohérence avec gsheet validé ✅

**Pull request merged**